js/mixins/uploader.js

```diff
@@ -222,7 +222,7 @@ export default {
          * See: http://docs.fineuploader.com/branch/master/features/drag-and-drop.html#processingDroppedFilesComplete
          */
         on_dropped_files_complete(files) {
-            if (this.canDrop) {
+            if (this.canDrop && this.$uploader) {
```

Comment:
what happens if `!this.$uploader`? drop cancels/fails? this might be hard to test though.

Reply:
on my last commit I got back to the original version, without the check against `this.$uploader` as it was before. 

Seems to work as it was, the other checks and changes I did seem to prevent `$uploader` from _not_ being set at this point now...